### ISSUE TITLE
Absence of parentNode causing issues

### ISSUE THREAD

**Issue by pankajpatel** [modulor-js/modulor-html#52]
https://github.com/modulor-js/modulor-html/blob/93a400d2b3387733138e5cada30ee3a1023c1e60/src/range.js#L26-L29

Due to race condition of UI initialization and User interaction, sometimes the parentNode does not exist and causes the append to fail.

**Comment by nogizhopaboroda** [comment]
Would simple presence of parentNode solve the issue? Like
```
 NodesRange.prototype.appendChild = function($el){ 
   this.stopNode.parentNode && this.stopNode.parentNode.insertBefore($el, this.stopNode); 
   this.update(); 
 };
```

**Comment by pankajpatel** [comment]
Yes it will; I was thinking about making a PR for this fix.

Though I am worried about how we can release this fix. This is needed on top of 1.2.3 rather than on 1.3.1? Is it possible?

As for adopting 1.3.1, our apps are breaking because of legacy way of creating and using components.

**Comment by nogizhopaboroda** [comment]
Oh, i didn't know about such issue with 1.3.1. Then you can create a branch out of this commit i guess https://github.com/modulor-js/modulor-html/commit/96f98959c785b25fcee08c2a30fb6da067fe5a54 , make the fix there and bump the minor version.

Is there any help i can provide with adopting 1.3.1?

**Comment by pankajpatel** [comment]
I need to check what can be done as quick fix for easy adoption of 1.3.1

Though I would do MR and use from git itself for now.

And we can merge the MR and keep the branch there for a while

**Comment by pankajpatel** [comment]
I think 1.3.1 can be adopted easily; I have fixed all the issues; creating an MR for this as well

can we release today @nogizhopaboroda ?
